libexec/scoop-import.ps1

```diff
@@ -50,7 +52,11 @@ foreach ($item in $import.apps) {
     }
 
     $app = if ($item.Source -in $bucket_names) {
-        "$($item.Source)/$($item.Name)"
+        if ($item.Version) {
```

Comment:
This check is redundant. `Version` is always present in the $item object.

Reply:
You are right, the version is inside the $item object. But I had the problem, that the version was ignored, when I imported a scoopfile with a version given in one of the apps. Maybe my expectation is wrong, but I want to install specific versions with a scoopfile in a project/repo. Therefore I wanted to give that version to scoop-install.ps1 some lines below this.

Reply:
If you specify a version in the `scoop-install` command, that version will get locked - and you will not be able to update the application. 

Specifying versions auto-generates the manifest, which is similar to case (2) in my above comment.

Reply:
Mmmmh, so I guess, my use case to install specific versions of a package with "scoop import" is not a valid one. Without this change it will not work.

Reply:
Correct. If you want to install specific versions, the machine you're exporting from should have those specific/pinned versions (i.e. the `<auto-generated>` source) in the first place.